waterfall.js

```diff
@@ -53,13 +50,23 @@
 	// Run script
 	$(function () {
         // Get the enabled state.
-        var backgroundPort = chrome.runtime.connect({name: "background"});
+        const backgroundPort = chrome.runtime.connect({name: "background"});
         backgroundPort.postMessage({cmd: "getEnabledState"});
         backgroundPort.onMessage.addListener(function(response) {
             setupEnabledSwitch(response, backgroundPort);
 
             // Only runs the entire extension if "enabledState" returns true
             if (!response) return;
+            
+            // Only run on reddit
+            if (window.location.href.indexOf("reddit") === -1 ) return;
```

Comment:
Would a "!" operator work just as well here?

Reply:
Truthy and falsey? No, because window.location.href.indexOf doesn't return a 0, it returns -1 which apparently counts as true.